src/types/ProposalFieldValue.ts

```diff
@@ -9,13 +10,19 @@ export interface ProposalFieldValue {
 	value: string;
 	createdAt: Date;
 	applicationFormField?: ApplicationFormField;
+	readonly isValid: boolean;
 }
 
 export type ProposalFieldValueWrite = Omit<
 	ProposalFieldValue,
 	'applicationFormField' | 'createdAt' | 'id' | 'proposalVersionId'
 >;
 
+type WritableProposalFieldValue = Writable<ProposalFieldValue>;
```

Comment:
I'm seeing why you used Partial earlier -- since you're using this pattern you need to mark non-writable fields as readonly in the base type (id, createdAt, and applicationFormField)